editor/app/fatalities/[record_locator]/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { useQuery } from "@/utils/graphql";
+import { GET_CRASH } from "@/queries/crash";
+import { Crash } from "@/types/crashes";
+import { useDocumentTitle } from "@/utils/documentTitle";
+import { formatAddresses } from "@/utils/formatters";
+import { formatYear } from "@/utils/formatters";
+
+export default function FatalCrashDetailsPage({
+  params,
+}: {
+  params: { record_locator: string };
+}) {
+  const recordLocator = params.record_locator;
+
+  const typename = "crashes";
+
+  const { data, error } = useQuery<Crash>({
+    query: recordLocator ? GET_CRASH : null,
+    variables: { recordLocator },
+    typename,
+  });
+
+  if (error) {
+    console.error(error);
+  }
+
+  // Set document title based on loaded data
+  useDocumentTitle(
+    data && data.length > 0
+      ? `Fatalities ${data[0].record_locator} - ${formatAddresses(data[0])}`
+      : "Vision Zero Editor",
+    true // exclude the suffix
+  );
+
+  return (
+    <>
+      {data && (
+        <div className="d-flex justify-content-between mb-3">
+          <span className="fs-3 fw-bold text-uppercase">
+            {formatAddresses(data[0])}
```

Comment:
Probably out of the scope of this issue, but I noticed that the code errors in the `formatAddress` call if I entered a non-existent crash record locator like http://localhost:3002/fatalities/123

Reply:
Right, so we can turn on the [`noUncheckedIndexedAccess`](https://www.typescriptlang.org/tsconfig/#noUncheckedIndexedAccess) compiler option if we want to check for this kind of error. 

Switching it on will require a bit of refactor across the app, because we will have to write some new type guards to deal with it. I'm going to open an issue for this!

Reply:
thanks for sharing and all of this is TIL! 🙏